gruntfile.js

```diff
@@ -140,19 +156,19 @@ module.exports = function (grunt) {
         'bower_components/moment/min/moment.min.js',
         'bower_components/xterm.js/dist/xterm.js',
         'bower_components/bootbox.js/bootbox.js',
-        'bower_components/toastr/toastr.min.js',
+        'assets/js/jquery.gritter.js', // Using custom version to fix error in minified build due to "use strict"
```

Comment:
You should merge latest develop branch into your repository first. toastr is now used instead of gritter.